src/Uno.UI/UI/Xaml/Controls/WebView/Core/CoreWebView2.Properties.cs

```diff
@@ -0,0 +1,91 @@
+﻿using Windows.Foundation;
+using Windows.UI.Xaml.Controls;
+
+namespace Microsoft.Web.WebView2.Core;
+#pragma warning disable CS0067 // TODO:MZ: Undo this
```

Comment:
@MartinZikmund Does this need to be addressed? Which event is not used?

Reply:
Yes, I will address it when implementing `WebView2` on some of the unsupported targets, but I think the warning is not actually happening anymore, the events in the file are now used 👍